bots/zephyr_mirror_backend.py

```diff
@@ -596,13 +622,14 @@ def zcrypt_encrypt_content(zephyr_class, instance, content):
     return encrypted
 
 def forward_to_zephyr(message):
+    # type: (Dict[str, Any]) -> Any
     support_heading = "Hi there! This is an automated message from Zulip."
     support_closing = """If you have any questions, please be in touch through the \
 Feedback button or at support@zulipchat.com."""
 
     wrapper = textwrap.TextWrapper(break_long_words=False, break_on_hyphens=False)
     wrapped_content = "\n".join("\n".join(wrapper.wrap(line))
-            for line in message["content"].replace("@", "@@").split("\n"))
+                                for line in message["content"].replace("@", "@@").split("\n"))
```

Comment:
Ah, sorry. There's just one line changed here. The 'f' in `for` lined up with the 'c' in `wrapped_content` in the revision before yours, and should be returned there.